BritneyS/DynamicTableView
# Add table view

## What does this PR do :question:
This PR represents the base project of an ongoing experiment to add different types of custom cells that can be shown or hidden based on available data.


## How to test it :microscope:
1. Check out this branch
2. Run the app
3. Note that a table view containing the words to the sentence `"The quick brown fox jumps over the lazy dog"` is shown (see screenshot)


## Any background info you would like to include :white_check_mark:
This project was started as an exploration for a Spike based on a sprint for the project https://github.com/detroit-labs/dl-ops-presentation-ios (private repo). 

The current PR doesn't demonstrate any final architectural decisions.


## Screenshots (if applicable) :camera:
![Simulator Screen Shot - iPhone XR - 2019-03-12 at 16 34 19](https://user-images.githubusercontent.com/8409475/54234406-6db09780-44e5-11e9-98bc-0ec3dff9d5aa.png)
